### PR TITLE
Nyt clone

### PR DESCRIPTION
**Pull request by NataJenkins**
New york times clone page using Html and css.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,554 @@
...
+                        than
+                        light and cooling to an even temperature as it expands. That is something like inflation. And
+                        the
+                        structure of the universe mirrors the froth and foam of the original pot.</p>
+                </div>
+            </div>
+        </div>
+        <hr>
+        <p class="larry">By LARRY BUCHANAN and JONATHAN CORUM</p>
+    </div>
```
Comment (SpaYco) [inline index.html:212]: - [ ] Do not use screenshots of the images, instead use one image just like the website (use the `position` property) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

File: index.html [hunk 2]
```diff
@@ -0,0 +1,554 @@
...
+    <script src="https://kit.fontawesome.com/7bc3c221d9.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <!-- NAVBAR -->
+    <nav class="navbar">
+        <div class="left">
+            <i class="icon fas fa-bars"></i>
+            <i class="icon fas fa-search"></i>
+            <p>SPACE & COSMOS</p>
```
Comment (SpaYco) [inline index.html]: - [ ] It should have a "Hamburger button", a search icon and a link with text on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))

- your hamburger and search icon should be a button

- the **SPACE & COSMOS** text should be a link

File: index.html [hunk 3]
```diff
@@ -0,0 +1,554 @@
...
+        </div>
+        <img src="./Images/Screenshot_25.jpg" alt="logo">
```
Comment (SpaYco) [inline index.html]: * [ ] The NY Times logo should be a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))

File: index.html [hunk 4]
```diff
@@ -0,0 +1,554 @@
...
+                    <div class="app-types">
+                        <p>Mobile Applications</p>
+                        <p>Replica Edition</p>
+                        <p>Espa;ol</p>
+                        <p>中文网</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
```
Comment (SpaYco) [inline index.html]: - [ ] all of these should be links ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello @NataJenkins! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 
 

- [ ] Your Demo Link is broken and shows a 404 error, please fix, could be that GitHub pages is deploying from the `master` branch instead of the `NyClone` one, if you couldn't find a fix, you can instead use https://raw.githack.com/

- [ ] Elements should be aligned with a combination of all three techniques ([float](https://css-tricks.com/all-about-floats/), [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [grid](https://css-tricks.com/snippets/css/complete-guide-grid/)). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_pathwright.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/)) and ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_odin.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements#introduction)), you're not using CSS Grid

- [ ] Files not related to the project should not be committed and added to .gitignore (e.g. `.app_screenshot.png`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- **I added more required changes down below**


## Optional 

- [ ] use the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

- [ ] style the social buttons instead of living them blue (if you're using font awesome use the `color` property


### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

File: index.html [hunk 5]
```diff
@@ -0,0 +1,557 @@
...
+    <script src="https://kit.fontawesome.com/7bc3c221d9.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <!-- NAVBAR -->
+    <nav class="navbar">
+        <div class="left">
+            <button class="icon-btn"><i class="icon fas fa-bars"></i></button>
+            <button class="icon-btn"><i class="icon fas fa-search"></i></button>
+            <a href="https://www.nytimes.com/section/science/space">SPACE & COSMOS</a>
```
Comment (expjazz) [inline index.html:18]: - [ ] The link should not be blue

- [ ] There is a white margin overflowing

| Yours | Original |
|:---:|:---:|
|image | image |

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Requires :recycle: 

Hey Team :+1: 

You did a great job on this project :clap: 

The page looks great, and you fixed most of what was pointed out by my colleague. We only have one small problem left on the navbar (check the inline comment below).

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Review by kubilaycaglayan — CHANGES_REQUESTED** [review]
Hello Team 👋 
Status: Changes Required ♻️

Great job so far 🌟 

- [ ]  Please try to fix the overflow one more time.

**HINT :** You have a "background-color: white" property on that part. You don't need it for this specific area.

**📸  Screenshot** => Notice the bottom line of the nav: it is being overflowed by the link container.
image

- [ ]  [OPTIONAL] Consider using **main** tag, it will make your code semantically better.

That's it. After doing the changes you can apply for another code review.
With all my respect... 🙏

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :clap: 

Great work on this project and on the changes :1st_place_medal: 

You can merge and move on to the next projects.

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**